### ISSUE TITLE
[BUG] HRRR - `AssertionError: Incorrect number of delay files written.`

### ISSUE THREAD

**Issue by cmarshak** [dbekaert/RAiDER#579]
**Describe the bug**
Here is the input job parameters and link to the log. Hoping this is transient, but cannot tell. This job failed twice. Would have to do more digging to determine why this job failed the first time *with same error*. Will circle back later. Please investigate.

```
{'job_id': '47b17e33-176b-48a7-afc1-4e264ade61bc',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 496,
   'granules': ['S1A_IW_SLC__1SDV_20171224T230634_20171224T230701_019851_021C5C_7C7C',
    'S1A_IW_SLC__1SDV_20171224T230659_20171224T230726_019851_021C5C_6B8B'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20170627T230631_20170627T230658_017226_01CBD2_A22F',
    'S1A_IW_SLC__1SDV_20170627T230656_20170627T230723_017226_01CBD2_2247'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/47b17e33-176b-48a7-afc1-4e264ade61bc/47b17e33-176b-48a7-afc1-4e264ade61bc.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [9096.196, 262.999]}
```
and here is the log:

```
 â•­â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â•®
 â”‚ I'm building Herbie's default config file.      â”‚
 â•°â•¥â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â•¯
 ðŸ‘·ðŸ»â€â™‚ï¸
 â•­â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â•®
 â”‚ You're ready to go.                             â”‚
 â”‚ You may edit the config file here:              â”‚
 â”‚ /home/raider/.config/herbie/config.toml         â”‚
 â•°â•¥â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â•¯
 ðŸ‘·ðŸ»â€â™‚ï¸
/opt/conda/envs/RAiDER/lib/python3.10/site-packages/RAiDER/cli/__main__.py:36: DeprecationWarning: Accessing entry points by index is deprecated. Cast to tuple if needed.
  process_entry_point = entry_points(group='console_scripts', name=f'{args.process}.py')[0]
Downloading s3://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf/47b17e33-176b-48a7-afc1-4e264ade61bc/S1-GUNW-A-R-004-tops-20171224_20170627-230700-00079W_00039N-PP-f37b-v3_0_0.nc to S1-GUNW-A-R-004-tops-20171224_20170627-230700-00079W_00039N-PP-f37b-v3_0_0.nc
Downloading s3://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf/47b17e33-176b-48a7-afc1-4e264ade61bc/S1-GUNW-A-R-004-tops-20171224_20170627-230700-00079W_00039N-PP-f37b-v3_0_0.json to S1-GUNW-A-R-004-tops-20171224_20170627-230700-00079W_00039N-PP-f37b-v3_0_0.json

Downloading products:   0%|          | 0/1 [00:00<?, ?product/s]

Downloading S1A_OPER_AUX_POEORB_OPOD_20210305T161134_V20171223T225942_20171225T005942.EOF:   0%|          | 0.00/4.41M [00:00<?, ?B/s][A

Downloading S1A_OPER_AUX_POEORB_OPOD_20210305T161134_V20171223T225942_20171225T005942.EOF:  24%|â–ˆâ–ˆâ–       | 1.05M/4.41M [00:00<00:02, 1.18MB/s][A

Downloading S1A_OPER_AUX_POEORB_OPOD_20210305T161134_V20171223T225942_20171225T005942.EOF:  48%|â–ˆâ–ˆâ–ˆâ–ˆâ–Š     | 2.10M/4.41M [00:01<00:00, 2.32MB/s][A

Downloading S1A_OPER_AUX_POEORB_OPOD_20210305T161134_V20171223T225942_20171225T005942.EOF:  95%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–Œ| 4.19M/4.41M [00:01<00:00, 4.88MB/s][A
Downloading S1A_OPER_AUX_POEORB_OPOD_20210305T161134_V20171223T225942_20171225T005942.EOF: 100%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ| 4.41M/4.41M [00:01<00:00, 3.73MB/s]

MD5 checksumming:   0%|          | 0.00/4.41M [00:00<?, ?B/s][A

                                                             [A
Downloading products: 100%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ| 1/1 [00:01<00:00,  1.82s/product]
Downloading products: 100%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ| 1/1 [00:01<00:00,  1.82s/product]
Wrote new cfg file: GUNW_20171224-20170627_230700.yaml
[33;21mWARNING: Weather model only extends to the surface topography; height levels below the topography will be interpolated from the surface and may be inaccurate.[0m
Invalid extension GTiff for cube. Defaulting to .nc
Invalid extension GTiff for cube. Defaulting to .nc
Output cube spacing: 0.05 degrees
Output SNWE: [38.55, 40.5, -79.35, -75.85]
Starting to run the weather model calculation
Date: 20171224
Beginning weather model pre-processing
Weather model HRRR is available from 2016-07-15 to Present
Weather model HRRR is available from 2016-07-15 to Present
âœ… Found â”Š model=hrrr â”Š [3mproduct=nat[0m â”Š [38;2;41;130;13m2017-Dec-24 23:00 UTC[92m F00[0m â”Š [38;2;255;153;0m[3mGRIB2 @ aws[0m â”Š [38;2;255;153;0m[3mIDX @ aws[0m
/opt/conda/envs/RAiDER/lib/python3.10/site-packages/herbie/core.py:797: UserWarning: This pattern is interpreted as a regular expression, and has match groups. To actually get the groups, use str.extract.
  logic = df.search_this.str.contains(searchString)
ðŸ‘¨ðŸ»â€ðŸ­ Created directory: [/home/raider/weather_files/hrrr/20171224]
Note: Returning a list of [12] xarray.Datasets because cfgrib opened with multiple hypercubes.
Number of weather model nodes: 904761
Shape of weather model: (111, 143, 57)
Bounds of the weather model: 128693.85/458693.85/1457479.86/1883479.86 (SNWE)
Weather model: HRRR
Mean value of the wet refractivity: 15.384904
Mean value of the hydrostatic refractivity: 183.989838
======Weather Model class object=====
Weather model time: 2017-12-24 23:00:00
Latitude resolution: 0.02702702702702703
Longitude resolution: 0.02702702702702703
Native projection: PROJCRS["unknown",BASEGEOGCRS["unknown",DATUM["unknown",ELLIPSOID["unknown",6371229,0,LENGTHUNIT["metre",1,ID["EPSG",9001]]]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8901]]],CONVERSION["unknown",METHOD["Lambert Conic Conformal (2SP)",ID["EPSG",9802]],PARAMETER["Latitude of false origin",38.5,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8821]],PARAMETER["Longitude of false origin",262.5,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8822]],PARAMETER["Latitude of 1st standard parallel",38.5,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8823]],PARAMETER["Latitude of 2nd standard parallel",38.5,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8824]],PARAMETER["Easting at false origin",0,LENGTHUNIT["metre",1],ID["EPSG",8826]],PARAMETER["Northing at false origin",0,LENGTHUNIT["metre",1],ID["EPSG",8827]]],CS[Cartesian,2],AXIS["(E)",east,ORDER[1],LENGTHUNIT["metre",1,ID["EPSG",9001]]],AXIS["(N)",north,ORDER[2],LENGTHUNIT["metre",1,ID["EPSG",9001]]]]
ZMIN: -100.0
ZMAX: 80000.0
k1 = 0.776
k2 = 0.233
k3 = 3750.0
Humidity type = q
=====================================
Class name: hrrr
Dataset: hrrr
=====================================
A: []
B: []
Number of points in Lon/Lat = 111/143
Total number of grid points (3D): 904761
=====================================
Extent of the weather model is (xmin, ymin, xmax, ymax):-80.67, 37.66, -75.00, 41.37
Extent of the input is (xmin, ymin, xmax, ymax): -79.35, 38.55, -75.85, 40.50
Weather model HRRR is available from 2016-07-15 to Present
Weather model HRRR is available from 2016-07-15 to Present
âœ… Found â”Š model=hrrr â”Š [3mproduct=nat[0m â”Š [38;2;41;130;13m2017-Dec-25 00:00 UTC[92m F00[0m â”Š [38;2;255;153;0m[3mGRIB2 @ aws[0m â”Š [38;2;255;153;0m[3mIDX @ aws[0m
/opt/conda/envs/RAiDER/lib/python3.10/site-packages/herbie/core.py:797: UserWarning: This pattern is interpreted as a regular expression, and has match groups. To actually get the groups, use str.extract.
  logic = df.search_this.str.contains(searchString)
ðŸ‘¨ðŸ»â€ðŸ­ Created directory: [/home/raider/weather_files/hrrr/20171225]
Note: Returning a list of [12] xarray.Datasets because cfgrib opened with multiple hypercubes.
Number of weather model nodes: 904761
Shape of weather model: (111, 143, 57)
Bounds of the weather model: 128693.85/458693.85/1457479.86/1883479.86 (SNWE)
Weather model: HRRR
Mean value of the wet refractivity: 16.064142
Mean value of the hydrostatic refractivity: 183.998825
======Weather Model class object=====
Weather model time: 2017-12-25 00:00:00
Latitude resolution: 0.02702702702702703
Longitude resolution: 0.02702702702702703
Native projection: PROJCRS["unknown",BASEGEOGCRS["unknown",DATUM["unknown",ELLIPSOID["unknown",6371229,0,LENGTHUNIT["metre",1,ID["EPSG",9001]]]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8901]]],CONVERSION["unknown",METHOD["Lambert Conic Conformal (2SP)",ID["EPSG",9802]],PARAMETER["Latitude of false origin",38.5,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8821]],PARAMETER["Longitude of false origin",262.5,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8822]],PARAMETER["Latitude of 1st standard parallel",38.5,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8823]],PARAMETER["Latitude of 2nd standard parallel",38.5,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8824]],PARAMETER["Easting at false origin",0,LENGTHUNIT["metre",1],ID["EPSG",8826]],PARAMETER["Northing at false origin",0,LENGTHUNIT["metre",1],ID["EPSG",8827]]],CS[Cartesian,2],AXIS["(E)",east,ORDER[1],LENGTHUNIT["metre",1,ID["EPSG",9001]]],AXIS["(N)",north,ORDER[2],LENGTHUNIT["metre",1,ID["EPSG",9001]]]]
ZMIN: -100.0
ZMAX: 80000.0
k1 = 0.776
k2 = 0.233
k3 = 3750.0
Humidity type = q
=====================================
Class name: hrrr
Dataset: hrrr
=====================================
A: []
B: []
Number of points in Lon/Lat = 111/143
Total number of grid points (3D): 904761
=====================================
Extent of the weather model is (xmin, ymin, xmax, ymax):-80.67, 37.66, -75.00, 41.37
Extent of the input is (xmin, ymin, xmax, ymax): -79.35, 38.55, -75.85, 40.50
/opt/conda/envs/RAiDER/lib/python3.10/site-packages/rasterio/__init__.py:304: NotGeoreferencedWarning: Dataset has no geotransform, gcps, or rpcs. The identity matrix will be returned.
  dataset = DatasetReader(path, driver=driver, sharing=sharing, **kwargs)
Processing slice 1 / 20: -500.0
Processing slice 2 / 20: 0.0
Processing slice 3 / 20: 500.0
Processing slice 4 / 20: 1000.0
Processing slice 5 / 20: 1500.0
Processing slice 6 / 20: 2000.0
Processing slice 7 / 20: 2500.0
Processing slice 8 / 20: 3000.0
Processing slice 9 / 20: 3500.0
Processing slice 10 / 20: 4000.0
Processing slice 11 / 20: 4500.0
Processing slice 12 / 20: 5000.0
Processing slice 13 / 20: 5500.0
Processing slice 14 / 20: 6000.0
Processing slice 15 / 20: 6500.0
Processing slice 16 / 20: 7000.0
Processing slice 17 / 20: 7500.0
Processing slice 18 / 20: 8000.0
Processing slice 19 / 20: 8500.0
Processing slice 20 / 20: 9000.0
[31;1mCRITICAL: There are missing delay values. Check your inputs.[0m
Successfully wrote delay cube to: /home/raider/HRRR_tropo_20171224T230709_ray.nc
Starting to run the weather model calculation
Date: 20170627
Beginning weather model pre-processing
Weather model HRRR is available from 2016-07-15 to Present
Weather model HRRR is available from 2016-07-15 to Present
ðŸ’” Did not find â”Š model=hrrr â”Š [3mproduct=nat[0m â”Š [38;2;41;130;13m2017-Jun-27 23:00 UTC[92m F00[0m
[31;21mERROR: 
No index file was found for None
Download the full file first (with `H.download()`).
You will need to remake the Herbie object (H = `Herbie()`)
or delete this cached property: `del H.index_as_dataframe()`[0m
[33;21mWARNING: [0m
Weather model HRRR is available from 2016-07-15 to Present
Weather model HRRR is available from 2016-07-15 to Present
ðŸ’” Did not find â”Š model=hrrr â”Š [3mproduct=nat[0m â”Š [38;2;41;130;13m2017-Jun-28 00:00 UTC[92m F00[0m
[31;21mERROR: 
No index file was found for None
Download the full file first (with `H.download()`).
You will need to remake the Herbie object (H = `Herbie()`)
or delete this cached property: `del H.index_as_dataframe()`[0m
[33;21mWARNING: [0m
[31;21mERROR: No weather model data was successfully processed.[0m
Traceback (most recent call last):
  File "/opt/conda/envs/RAiDER/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/conda/envs/RAiDER/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/opt/conda/envs/RAiDER/lib/python3.10/site-packages/RAiDER/cli/__main__.py", line 46, in <module>
    main()
  File "/opt/conda/envs/RAiDER/lib/python3.10/site-packages/RAiDER/cli/__main__.py", line 42, in main
    process_entry_point.load()()
  File "/opt/conda/envs/RAiDER/lib/python3.10/site-packages/RAiDER/cli/raider.py", line 535, in calcDelaysGUNW
    assert len(cube_filenames) == 2, 'Incorrect number of delay files written.'
AssertionError: Incorrect number of delay files written.
```

**Comment by cmarshak** [comment]
It is occuring frequently. Here are 10 jobs with the same error.

```
{'job_id': '47b17e33-176b-48a7-afc1-4e264ade61bc',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 496,
   'granules': ['S1A_IW_SLC__1SDV_20171224T230634_20171224T230701_019851_021C5C_7C7C',
    'S1A_IW_SLC__1SDV_20171224T230659_20171224T230726_019851_021C5C_6B8B'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20170627T230631_20170627T230658_017226_01CBD2_A22F',
    'S1A_IW_SLC__1SDV_20170627T230656_20170627T230723_017226_01CBD2_2247'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/47b17e33-176b-48a7-afc1-4e264ade61bc/47b17e33-176b-48a7-afc1-4e264ade61bc.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [9096.196, 262.999]},
 {'job_id': '2060fca1-fdfa-420b-a995-200d47d65752',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 492,
   'granules': ['S1A_IW_SLC__1SDV_20170627T230516_20170627T230543_017226_01CBD2_8305',
    'S1A_IW_SLC__1SDV_20170627T230540_20170627T230608_017226_01CBD2_A473'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161217T230535_20161217T230605_014426_017645_64C0'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/2060fca1-fdfa-420b-a995-200d47d65752/2060fca1-fdfa-420b-a995-200d47d65752.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [8134.213, 254.858]},
 {'job_id': 'eba02cab-2de0-4b3a-a168-03e5c650c7eb',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 498,
   'granules': ['S1A_IW_SLC__1SDV_20170627T230721_20170627T230748_017226_01CBD2_F054',
    'S1A_IW_SLC__1SDV_20170627T230746_20170627T230813_017226_01CBD2_C7FC'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161229T230718_20161229T230745_014601_017BBF_1871',
    'S1A_IW_SLC__1SDV_20161229T230743_20161229T230810_014601_017BBF_C530'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/eba02cab-2de0-4b3a-a168-03e5c650c7eb/eba02cab-2de0-4b3a-a168-03e5c650c7eb.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [10346.334, 256.427]},
 {'job_id': '03fd159d-ad49-4b34-a9cb-05cd5637d85d',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 497,
   'granules': ['S1A_IW_SLC__1SDV_20170627T230656_20170627T230723_017226_01CBD2_2247',
    'S1A_IW_SLC__1SDV_20170627T230721_20170627T230748_017226_01CBD2_F054'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161217T230653_20161217T230720_014426_017645_C800',
    'S1A_IW_SLC__1SDV_20161217T230718_20161217T230745_014426_017645_0C22'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/03fd159d-ad49-4b34-a9cb-05cd5637d85d/03fd159d-ad49-4b34-a9cb-05cd5637d85d.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [10246.054, 243.402]},
 {'job_id': '903b32d8-8a67-4285-955c-e3ad82cee737',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 499,
   'granules': ['S1A_IW_SLC__1SDV_20170627T230746_20170627T230813_017226_01CBD2_C7FC',
    'S1A_IW_SLC__1SDV_20170627T230811_20170627T230836_017226_01CBD2_2149'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161217T230743_20161217T230810_014426_017645_D2E6',
    'S1A_IW_SLC__1SDV_20161217T230808_20161217T230834_014426_017645_5A20'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/903b32d8-8a67-4285-955c-e3ad82cee737/903b32d8-8a67-4285-955c-e3ad82cee737.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [9648.763, 251.675]},
 {'job_id': '1b89a7c5-57ee-46ae-ae3c-a30e12898440',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 493,
   'granules': ['S1A_IW_SLC__1SDV_20170802T230542_20170802T230610_017751_01DBC4_2A71',
    'S1A_IW_SLC__1SDV_20170802T230608_20170802T230635_017751_01DBC4_3FD2'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20170627T230540_20170627T230608_017226_01CBD2_A473',
    'S1A_IW_SLC__1SDV_20170627T230606_20170627T230633_017226_01CBD2_69FA'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/1b89a7c5-57ee-46ae-ae3c-a30e12898440/1b89a7c5-57ee-46ae-ae3c-a30e12898440.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [8488.103, 250.664]},
 {'job_id': '5a9687ea-e48e-4cc2-a13f-a89b092cc84b',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 493,
   'granules': ['S1A_IW_SLC__1SDV_20170627T230540_20170627T230608_017226_01CBD2_A473',
    'S1A_IW_SLC__1SDV_20170627T230606_20170627T230633_017226_01CBD2_69FA'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20170603T230539_20170603T230607_016876_01C11A_4C22',
    'S1A_IW_SLC__1SDV_20170603T230605_20170603T230632_016876_01C11A_F777'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/5a9687ea-e48e-4cc2-a13f-a89b092cc84b/5a9687ea-e48e-4cc2-a13f-a89b092cc84b.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [8298.501, 264.706]},
 {'job_id': '07fcb6a6-96ee-4272-a1bd-eefd548e40ee',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 499,
   'granules': ['S1A_IW_SLC__1SDV_20170627T230746_20170627T230813_017226_01CBD2_C7FC',
    'S1A_IW_SLC__1SDV_20170627T230811_20170627T230836_017226_01CBD2_2149'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161229T230743_20161229T230810_014601_017BBF_C530',
    'S1A_IW_SLC__1SDV_20161229T230807_20161229T230834_014601_017BBF_907F'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/07fcb6a6-96ee-4272-a1bd-eefd548e40ee/07fcb6a6-96ee-4272-a1bd-eefd548e40ee.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [9680.452, 254.439]},
 {'job_id': '71d0c47f-b54c-4792-a0e8-89148f0df8fc',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 498,
   'granules': ['S1A_IW_SLC__1SDV_20170627T230721_20170627T230748_017226_01CBD2_F054',
    'S1A_IW_SLC__1SDV_20170627T230746_20170627T230813_017226_01CBD2_C7FC'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20170522T230718_20170522T230745_016701_01BBB0_EF14',
    'S1A_IW_SLC__1SDV_20170522T230743_20170522T230811_016701_01BBB0_9592'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/71d0c47f-b54c-4792-a0e8-89148f0df8fc/71d0c47f-b54c-4792-a0e8-89148f0df8fc.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [7699.378, 249.464]},
 {'job_id': '61136c47-7f9d-4f0b-9eba-9bd536755cba',
  'job_type': 'INSAR_ISCE_TEST',
  'request_time': '2023-08-21T23:42:11+00:00',
  'status_code': 'FAILED',
  'user_id': 'cmarshak',
  'name': 'track4-n3_resubmit1',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 497,
   'granules': ['S1A_IW_SLC__1SDV_20170721T230657_20170721T230724_017576_01D66A_9D8C',
    'S1A_IW_SLC__1SDV_20170721T230722_20170721T230749_017576_01D66A_94E9'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20170627T230656_20170627T230723_017226_01CBD2_2247',
    'S1A_IW_SLC__1SDV_20170627T230721_20170627T230748_017226_01CBD2_F054'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/61136c47-7f9d-4f0b-9eba-9bd536755cba/61136c47-7f9d-4f0b-9eba-9bd536755cba.log'],
  'expiration_time': '2023-10-22T00:00:00+00:00',
  'processing_times': [9301.677, 259.224]}]
```

**Comment by cmarshak** [comment]
https://github.com/dbekaert/RAiDER/blob/dev/test/gunw_azimuth_test_data/S1-GUNW-A-R-064-tops-20210723_20210711-015000-00119W_00033N-PP-6267-v2_0_6.nc

**Comment by bbuzz31** [comment]
All these fail because there is no HRRR data available from the server on 20170627
